src/azure-cli/azure/cli/command_modules/rdbms/_help.py

```diff
@@ -1221,3 +1221,19 @@
     type: command
     short-summary: Show the connection strings for a PostgreSQL flexible-server database.
 """
+
```

Comment:
If you add for this command for MariaDb, update this as well

Reply:
Same!